aidants_connect_web/tests/test_views/test_service.py

```diff
@@ -326,12 +326,9 @@ def test_stats_url_triggers_the_correct_template(self):
         self.assertTemplateUsed(response, "public_website/guide_utilisation.html")
 
 
-@tag("service")
+@tag("service", "faq")
 class FAQTests(TestCase):
-    def test_faq_url_triggers_the_correct_view(self):
-        found = resolve("/faq/")
-        self.assertEqual(found.func, service.faq_generale)
```

Comment:
Tu peux remplacer par :

```suggestion
        self.assertEqual(found.func.view_class, service.FaqCategoryView)
```